code/modules/mob/living/carbon/human/species_types/syndicate_supersoldier.dm

```diff
@@ -0,0 +1,19 @@
+/datum/species/human/supersoldier_syndie
```

Comment:
god, just put this in the supersoldier file and make it a subtype, dammit

Reply:
i just did